src/export.js

```diff
@@ -47,8 +47,11 @@ Object.defineProperty( QUnit, "reset", {
 } );
 
 if ( defined.document ) {
-	if ( window.QUnit ) {
-		throw new Error( "QUnit has already been defined." );
+	if ( window.QUnit && window.QUnit.test ) {
```

Comment:
Would prefer that we check `window.QUnit.version`. I don't see `test` going away any time soon, but I think that checking `version` (which is essentially just a tag) makes more sense as it is unlikely to ever be mocked out.